pkg/fanal/types/artifact.go

```diff
@@ -203,6 +206,9 @@ type ArtifactReference struct {
 
 	// SBOM
 	CycloneDX *CycloneDX
+
+	// Files provided by the package
+	SystemInstalledFiles []string `json:",omitempty"`
```

Comment:
Why do we need this field?

Reply:
Good catch! It was added by mistake. It will be fixed along with the other comment in the next commit.